release_notes/USING_HDF4_CMake.txt

```diff
@@ -29,25 +29,38 @@ Notes: This short instruction is written for users who want to quickly
             of using a ctest script for building and testing. See
             INSTALL_CMake.txt for more information.
 
+         4. See https://cmake.org/cmake/help/latest/command/find_package.html
+            for more information on the CMake "Config Mode Search Procedure".
 
 
 ========================================================================
 I. Preconditions
 ========================================================================
 
-   1. We suggest you obtain the latest CMake for windows from the Kitware
-      web site. The HDF 4.2.x product requires a minimum CMake version
-      of 3.10.1.
+   1. We suggest you obtain the latest CMake for your platform from the Kitware
+      web site. The HDF-4.2.x product requires a minimum CMake version
```

Comment:
Is '-' intentional?  Isn't HDF 4.2.x the way we normally use?

Reply:
okay